### ISSUE TITLE
Option with more than 2 names not showing all names

### ISSUE THREAD

**Issue by mrvkino** [ewels/rich-click#32]
Hi,

not sure if this is a bug or an intended feature.
I can see why when formatting multiple long name and short name can be a nightmare to support.

If we run the following code, only the short name version is displayed.

```python
import rich_click as click

@click.command()
@click.option(
    "-l",
    "--long",
    "--long-argument",
    help="""Long argument with multiple name.""",
)
def main(long_argument) -> None:
    print(long_argument)


if __name__ == "__main__":
    main()
```

<img width="1061" alt="image" src="https://user-images.githubusercontent.com/279511/156284691-f3e4d4d3-a5bd-4d1f-bf67-1887a2218e7a.png">

It will only show the first name in the list (in this case the `-l`).

**Comment by ewels** [comment]
Ok, I've rewritten the code that handles the options as it was pretty nasty and limited to only two options.

Because the output uses a table with columns, I didn't want to add a new column for each option as that could get messy fast. So now, all long form options (`--foo`) are shown in the first column, separated by commas, and all short-form options (`-f`)  are in the second.

<img width="917" alt="image" src="https://user-images.githubusercontent.com/465550/156349732-d90e1e57-f17b-4222-8981-1b95fcda31b8.png">

Hopefully this should handle any number of options but keep the display rigid enough to look good in most cases.

Thanks for reporting this!

**Comment by ewels** [comment]
Published in v1.2.1 https://pypi.org/project/rich-click/1.2.1/
